src/app/cube/browse/browse.component.ts

```diff
@@ -300,6 +302,7 @@ export class BrowseComponent implements AfterViewInit, OnDestroy {
     event.stopPropagation();
     this.query.orderBy = undefined;
     this.query.sortType = undefined;
+    this.sortText = '';
```

Comment:
```suggestion
    this.sortText = 'Date (DESC)';
```
This is technically always the default. This would still allow the dropdown UI to be highlighted

Reply:
Sorry, I had a misunderstanding at Standup, you're correct